eox_core/api/task_dispatcher/tests/test_views.py

```diff
@@ -0,0 +1,118 @@
+"""
+Test module for the Task dispatcher API
+"""
+from django.contrib.auth.models import User
+from django.test import TestCase, override_settings
+from django.urls import reverse
+from mock import MagicMock, patch
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from eox_core.api.task_dispatcher.v1.views import TaskAPI
+
+
+@override_settings(EOX_CORE_ASYNC_TASKS=["dummy.task.path.task_name"])
+class TestTaskAPI(TestCase):
+    """ Tests for the task dispatcher API """
+
+    patch_permissions = patch(
+        "rest_framework.permissions.IsAdminUser.has_permission",
+        return_value=True,
+    )
+
+    def setUp(self):
+        """setup fixtures"""
+        self.api_user = User(1, "test@example.com", "test")
+        self.client = APIClient()
+        self.client.force_authenticate(user=self.api_user)
+        self.url = reverse("eox-task-api:task-api")
+
+    @patch("celery.current_app.AsyncResult")
+    @patch_permissions
+    def test_get_task_statuses(self, _, m_async_result):
+        """Test that the GET method works with default parameters"""
+
```

Comment:
Can you remove this line?

Try to use this way of grouping inside a test method: Arrange Act Assert 

https://www.c-sharpcorner.com/UploadFile/dacca2/fundamental-of-unit-testing-understand-aaa-in-unit-testing/